<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChooseLoadingModeDialog</class>
 <widget class="QDialog" name="ChooseLoadingModeDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>314</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Loading Mode</string>
  </property>
  <widget class="QPushButton" name="ok_button">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>97</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>245</width>
     <height>69</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QRadioButton" name="points_radio">
    <property name="text">
     <string>Points in color</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="triangles_radio">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>Triangles - monochrome</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="triangles_points_radio">
    <property name="text">
     <string>Triangle (mono) + points (color)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
